contracts/openzeppelin/Ownable.sol

```diff
@@ -40,13 +42,43 @@ contract Ownable is Context {
         _;
     }
 
+    /**
+     * @dev Throws if called by any account which is not one of the administers.
+     */
+    modifier onlyAdmin() {
+        require(isAdmin(), "Ownable: caller is not one of the administers");
```

Comment:
seems it should be `administrators` or  `admins` in the error message, instead of `administers`😂